luarules/gadgets/unit_shield_behaviour.lua

```diff
@@ -1,12 +1,14 @@
 local gadget = gadget ---@type Gadget
 
+local reworkEnabled = Spring.GetModOptions().shieldsrework --remove when shield rework is permanent
+
 function gadget:GetInfo()
 	return {
 		name = "Shield Behaviour",
 		desc = "Overrides default shield engine behavior.",
 		author = "SethDGamre",
 		layer = 1,
-		enabled = true
+		enabled = reworkEnabled,
```

Comment:
Might as well put `Spring.GetModOptions().shieldsrework` here and remove the var since all its uses below are redundant if the whole gadget is disabled. On the other hand this whole file seems to be a pile of shitcode so maybe that's the least of its issues.